File: app/src/main/java/com/example/hi/MainActivity.java
package com.example.hi;
import android.app.Activity; import android.os.Bundle; import android.graphics.Color; import android.view.Gravity; import android.widget.TextView;
public class MainActivity extends Activity { public void onCreate(Bundle s){super.onCreate(s); TextView t=new TextView(this); t.setText("سلام"); t.setTextSize(32); t.setTextColor(Color.BLACK); t.setGravity(Gravity.CENTER); setContentView(t);} }

File: app/src/main/java/com/example/hi/MainActivity.kt
package com.example.hi

import android.app.Activity
import android.graphics.Color
import android.os.Bundle
import android.view.Gravity
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(state: Bundle?) {
        super.onCreate(state)
        val text = TextView(this).apply {
            text = "سلام"
            textSize = 32f
            setTextColor(Color.BLACK)
            gravity = Gravity.CENTER
        }
        setContentView(text)
    }
}
